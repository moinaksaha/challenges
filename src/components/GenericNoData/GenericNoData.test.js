import React from 'react'
import { shallow } from 'enzyme';
import Genericnodata from './index.js'
import toJson from 'enzyme-to-json';

describe('Component: <Genericnodata />', () => {

  test('Render Status: Success', () => {
    const wrapper = shallow(<Genericnodata />);
    expect(wrapper.exists()).toBe(true);
  })

  test('Snapshot: Enzyme', () => {
    const tree = shallow(<Genericnodata />)
    expect(toJson(tree)).toMatchSnapshot()
  })
  
})