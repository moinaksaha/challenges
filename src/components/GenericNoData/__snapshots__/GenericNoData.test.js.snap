// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Genericnodata /> Snapshot: Enzyme 1`] = `
<styled.div>
  <h3>
    Scary story!
  </h3>
  <p>
    No data found to display
  </p>
</styled.div>
`;
